jest.config.js

```diff
@@ -41,10 +41,10 @@ module.exports = {
   // An object that configures minimum threshold enforcement for coverage results
   coverageThreshold: {
     global: {
-      branches: 100,
```

Comment:
I feel that we should try to keep coverage for new libraries at 100% as much as possible. It's true we have libraries/packages where these numbers are lower than 100, but that is because coverage was added after tests were already written. Our plan has always been to bring them back up over time, so that is some tech debt that we have to pay back. However, we should try not to intentionally introduce tech debt if we can avoid it. Is it possible to increase the coverage?